extensions/amp-ad/0.1/test/test-amp-ad-api-handler.js

```diff
@@ -65,17 +68,80 @@ describe('amp-ad-api-handler', () => {
       }));
     });
 
-    it('should resolve startUp() when render-start API is called', () => {
-      expect(iframe.style.visibility).to.equal('hidden');
-      iframe.postMessageToParent({
-        sentinel: 'amp3ptest' + testIndex,
-        type: 'render-start',
+    describe('should resolve startUp', () => {
+      it('if render-start NOT supported', () => {
+        expect(iframe.style.visibility).to.equal('hidden');
+        iframe.postMessageToParent({
+          sentinel: 'amp3ptest' + testIndex,
+          type: 'bootstrap-loaded',
+        });
+        return startUpPromise.then(() => {
+          expect(iframe.style.visibility).to.equal('');
+        });
       });
-      return startUpPromise.then(() => {
-        expect(iframe.style.visibility).to.equal('');
+
+      it('if render-start supported, receive render-start first', () => {
+        adImpl.adType = 'doubleclick';
+        apiHandler = new AmpAdApiHandler(adImpl, adImpl.element);
+        apiHandler.noContentCallback_ = () => {};
```

Comment:
please avoid accessing private member in test. in this case, you can pass it in via the constructor in the previous line